core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
I think any fix should target `wh_media_map:init_map/0` and its execution, as I would expect that to be the cause of this issue.

The whole point of building the mapping as a cache is to avoid querying the database over and over for a prompt that may not exist (perhaps it was improperly imported, or the code referencing the prompt was typed in wrong). So I don't know that code should 'assume' a prompt exists if it is missing from the ETS table.